docs/src/pages/[platform]/connected-components/authenticator/quick-start-pull.android.mdx

```diff
@@ -0,0 +1,19 @@
+import { Link } from '@aws-amplify/ui-react';
+
+Then, run `amplify add auth` to configure how you want the Authenticator and its associated Cognito resources to behave. If you want more details on all of the options available in this step, please check out the Amplify Android Auth [getting started page](https://docs.amplify.aws/lib/auth/getting-started/q/platform/android/#set-up-backend-resources) or the more detailed Amplify CLI [documentation](https://docs.amplify.aws/cli/auth/overview/).
+
+```shell
+amplify add auth
+  ...
```

Comment:
```suggestion
```

Should this ... be here?

Reply:
yeah it’s implying the CLI steps, it could say `[CLI steps here]` instead for example